### ISSUE TITLE
Crashing Mac Big Sur

### ISSUE THREAD

**Issue by seanbischoff** [microsoft/vscode#145258]
Issue Type: <b>Bug</b>

When working with VSCode, NGINX & Node I am getting frequent crashes of my Mac. This did not happen prior to Big Sur. It just crashed again today.

VS Code version: Code 1.65.2 (c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1, 2022-03-10T14:33:49.188Z)
OS version: Darwin x64 20.6.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|14, 33, 18|
|Memory (System)|32.00GB (13.65GB free)|
|Process Argv|--crash-reporter-id 1ad11934-8490-4aac-beab-f552ee619296|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
jslint|ajh|1.2.1
project-manager|ale|12.5.0
atlascode|atl|2.10.9
vscode-custom-css|be5|6.0.2
braze-liquid-preview|chi|0.1.11
gitignore|cod|0.7.0
bracket-pair-colorizer|Coe|1.0.62
git-extension-pack|don|0.1.3
githistory|don|0.6.19
gitlens|eam|12.0.4
prettier-vscode|esb|9.3.0
pretty-formatter|mbl|0.2.2
rainbow-csv|mec|2.0.0
diff-merge|mos|0.6.1
vscode-docker|ms-|1.20.0
vsliveshare|ms-|1.0.5449
indent-rainbow|ode|8.2.2
java|red|1.4.0
vscode-commons|red|0.0.6
vscode-yaml|red|1.5.1
shopify-liquid|sis|2.3.0
open-in-browser|tec|2.0.0
vscodeintellicode|Vis|1.2.17
vscode-java-debug|vsc|0.38.0
vscode-java-dependency|vsc|0.19.0
vscode-java-pack|vsc|0.22.0
vscode-java-test|vsc|0.34.2
vscode-maven|vsc|0.35.1
vscode-todo-highlight|way|1.0.5
local-history|xyz|1.8.1
vscode-open-in-github|ziy|1.3.6

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vsclayoutctrt:30451275
dsvsc009:30452663
pythonvsnew555:30450902
vscscmwlcmc:30438804
pynewfile477:30450038

```

</details>

**Comment by mjbvz** [comment]
Is there anything specific that triggers the crash?

Also does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by seanbischoff** [comment]
It was building up to a repro with all extensions disabled, e.g. the fan started spinning up, a precursor.

I've installed the Insiders build for iOS and so far that fan has not spun up. I need to give it another hour or so to be sure.

I'll post an update tomorrow.

**Comment by deepak1556** [comment]
If the crash happens with insiders can you share the crash logs using the steps from https://github.com/microsoft/vscode/wiki/Native-Crash-Issues#creating-and-symbolicating-local-crash-reports, thanks!

**Comment by seanbischoff** [comment]
Insiders was better, but after about 2 hours today, no extensions installed, CPU overloaded and had to hard restart. I'll share the crash & also the electron-minidump reports.

**Comment by seanbischoff** [comment]
Happened again. No crash report because I had to hard-stop restart the laptop. Activity Monitor showed no CPU spike. I observed Insiders Source Control was timing out.
![vscode-insider-iOS-11 6 5](https://user-images.githubusercontent.com/4175169/159097004-3d6a06bc-00b1-4012-9375-5b2163fd7bce.jpg)

**Comment by seanbischoff** [comment]
Closing this ticket. Thanks for the support! Our IT department figured out the issue and it's unrelated to VSCode. It turned out that MDM profiles were not allowing our security software in the Mac Security Settings. Adding them resolved the issue for me in Mac 11.6.5. A coworker is still experiencing freeze issues, but he's not using VSCode and is on Mac OS 12.
